### PR TITLE
Facelift: Localization and Plugin POM

### PR DESCRIPTION
**Pull request by oleg-nenashev**
I was experimenting with PCT runs in Docker, and finally I discovered that the pom.xml is quite behind the recent versions. hence the update

* Some assistive updates towards https://issues.jenkins-ci.org/browse/JENKINS-55099
* Localization cleanup - https://issues.jenkins-ci.org/browse/JENKINS-55292

Depends on https://github.com/jenkinsci/plugin-pom/pull/143

@jenkinsci/java11-support

### CODE REVIEW

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties [hunk 1]
```diff
@@ -20,16 +20,16 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-MailSender.ListEmpty=Der Versuch wurde ignoriert, eine E-Mail an eine leere Liste von Empf�ngern zu verschicken.
-MailSender.NoAddress=Es konnte keine E-Mail an {0} geschickt werden, weil die E-Mail-Adresse unbekannt ist und kein Standardwert f�r die E-Mail-Domain eingestellt ist.
+MailSender.ListEmpty=Der Versuch wurde ignoriert, eine E-Mail an eine leere Liste von Empf�ngern zu verschicken.
```
Comment (alecharp) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties]: It seems to be a mistake

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties [hunk 2]
```diff
@@ -20,16 +20,16 @@
...
+MailSender.NoAddress=Es konnte keine E-Mail an {0} geschickt werden, weil die E-Mail-Adresse unbekannt ist und kein Standardwert f�r die E-Mail-Domain eingestellt ist.
 MailSender.BackToNormal.Normal=normal
 MailSender.BackToNormal.Stable=stabil
 MailSender.BackToNormalMail.Subject=Jenkins-Build ist wieder {0}:
 MailSender.UnstableMail.Subject=Jenkins-Build ist instabil:
 MailSender.UnstableMail.ToUnStable.Subject=Jenkins-Build ist instabil geworden: 
 MailSender.UnstableMail.StillUnstable.Subject=Jenkins-Build ist immer noch instabil:
 MailSender.FailureMail.Subject=Jenkins-Build fehlgeschlagen:
-MailSender.FailureMail.Changes=�nderungen:
+MailSender.FailureMail.Changes=�nderungen:
```
Comment (alecharp) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties]: Mistake again I guess

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 3]
```diff
@@ -20,20 +20,22 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-MailSender.ListEmpty=Ignorado un intento de envio de correos a una lista de destinatarios vac�a.
-MailSender.NoAddress=Fall� el envio de correo a {0} porque ninguna direcci�n de correo es conocida, y no se ha configurado ning�n dominio por defecto.
+MailSender.ListEmpty=Ignorado un intento de envio de correos a una lista de destinatarios vac�a.
```
Comment (alecharp) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]: same

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 4]
```diff
@@ -20,20 +20,22 @@
...
+MailSender.NoAddress=Fall� el envio de correo a {0} porque ninguna direcci�n de correo es conocida, y no se ha configurado ning�n dominio por defecto.
```
Comment (alecharp) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]: same

**Comment by oleg-nenashev** [comment]
This is how my IDE works :( will try to revert

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties (hunk 1 above)
Comment (oleg-nenashev) [reply src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties]:
```suggestion
MailSender.ListEmpty=Der Versuch wurde ignoriert, eine E-Mail an eine leere Liste von Empfängern zu verschicken.
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties (hunk 2 above)
Comment (oleg-nenashev) [reply src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties]:
```suggestion
MailSender.FailureMail.Changes=Änderungen:
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties (hunk 3 above)
Comment (oleg-nenashev) [reply src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
MailSender.ListEmpty=Ignorado un intento de envio de correos a una lista de destinatarios vacía.
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties (hunk 4 above)
Comment (oleg-nenashev) [reply src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
MailSender.NoAddress=Falló el envio de correo a {0} porque ninguna dirección de correo es conocida, y no se ha configurado ningún dominio por defecto.
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 5]
```diff
@@ -20,20 +20,22 @@
...
 MailSender.BackToNormal.Normal=Normal
 MailSender.BackToNormal.Stable=Estable
 MailSender.BackToNormalMail.Subject=El estado de Jenkins ha vuelto a {0} :
-MailSender.UnstableMail.Subject=El resultado de la ejecuci�n es inestable:
+MailSender.UnstableMail.Subject=El resultado de la ejecuci�n es inestable:
```
Comment (oleg-nenashev) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
MailSender.UnstableMail.Subject=El resultado de la ejecución es inestable:
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 6]
```diff
@@ -20,20 +20,22 @@
...
 MailSender.UnstableMail.ToUnStable.Subject=El estado de Jenkins es inestable: 
-MailSender.UnstableMail.StillUnstable.Subject=El estado de Jenkins contin�a siendo inestable:
-MailSender.FailureMail.Subject=La ejecuci�n en Jenkins ha fallado:
+MailSender.UnstableMail.StillUnstable.Subject=El estado de Jenkins contin�a siendo inestable:
```
Comment (oleg-nenashev) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
MailSender.UnstableMail.StillUnstable.Subject=El estado de Jenkins continúa siendo inestable:
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 7]
```diff
@@ -20,20 +20,22 @@
...
+MailSender.FailureMail.Subject=La ejecuci�n en Jenkins ha fallado:
```
Comment (oleg-nenashev) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
MailSender.FailureMail.Subject=La ejecución en Jenkins ha fallado:
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 8]
```diff
@@ -20,20 +20,22 @@
...
 MailSender.FailureMail.Changes=Cambios:
-MailSender.FailureMail.FailedToAccessBuildLog=Error al acceder al "log" de ejecuci�n
+MailSender.FailureMail.FailedToAccessBuildLog=Error al acceder al "log" de ejecuci�n
```
Comment (oleg-nenashev) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
MailSender.FailureMail.FailedToAccessBuildLog=Error al acceder al "log" de ejecución
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 9]
```diff
@@ -20,20 +20,22 @@
...
 MailSender.Link=Echa un vistazo a <{0}{1}>
 
-Mailer.DisplayName=Notificaci�n por correo
+Mailer.DisplayName=Notificaci�n por correo
```
Comment (oleg-nenashev) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
Mailer.DisplayName=Notificación por correo
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 10]
```diff
@@ -20,20 +20,22 @@
...
 Mailer.UserProperty.DisplayName=E-mail 
 Mailer.Unknown.Host.Name=Nombre de Host desconocido:
-Mailer.Suffix.Error=Este campo deberia ser el s�mbolo "@" seguido de un nombre de dominio.
+Mailer.Suffix.Error=Este campo deberia ser el s�mbolo "@" seguido de un nombre de dominio.
```
Comment (oleg-nenashev) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
Mailer.Suffix.Error=Este campo deberia ser el símbolo "@" seguido de un nombre de dominio.
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties [hunk 11]
```diff
@@ -20,20 +20,22 @@
...
+
+MailCommand.ShortDescription=Lee de la entrada est�ndard, y lo env�a por email.
```
Comment (oleg-nenashev) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_es.properties]:
```suggestion
MailCommand.ShortDescription=Lee de la entrada estándard, y lo envía por email.
```

File: src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties [hunk 12]
```diff
@@ -20,16 +20,16 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-MailSender.ListEmpty=Der Versuch wurde ignoriert, eine E-Mail an eine leere Liste von Empf�ngern zu verschicken.
-MailSender.NoAddress=Es konnte keine E-Mail an {0} geschickt werden, weil die E-Mail-Adresse unbekannt ist und kein Standardwert f�r die E-Mail-Domain eingestellt ist.
+MailSender.ListEmpty=Der Versuch wurde ignoriert, eine E-Mail an eine leere Liste von Empfängern zu verschicken.
+MailSender.NoAddress=Es konnte keine E-Mail an {0} geschickt werden, weil die E-Mail-Adresse unbekannt ist und kein Standardwert f�r die E-Mail-Domain eingestellt ist.
```
Comment (alecharp) [inline src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties]: Still a problem here.

Comment (oleg-nenashev) [reply src/main/resources/jenkins/plugins/mailer/tasks/i18n/Messages_de.properties]:
```suggestion
MailSender.NoAddress=Es konnte keine E-Mail an {0} geschickt werden, weil die E-Mail-Adresse unbekannt ist und kein Standardwert für die E-Mail-Domain eingestellt ist.
```

**Pull request merged**